Matlab.tmbundle/Syntaxes/MATLAB.tmLanguage

```diff
@@ -1463,7 +1463,7 @@
 									<key>contentName</key>
 									<string>meta.assignment.variable.output.matlab</string>
 									<key>begin</key>
-									<string>\G(?=.*?=)</string>
+									<string>\G(?=[^\(]*?(?:=|\[|\.{3}))</string>
```

Comment:
The previous regex expected `=` on the same line, which does not allow for line continuations between the output arguments and `=`.

In order to differentiate between a function name and a single output argument, the negative lookforward on `[^\(]*` expects a parathesis open `(` on the same line as the function name. This determines the behavior of option 2 in the PR description.